class/practice_10.1/match.cpp

```diff
@@ -0,0 +1,24 @@
+#include "match.hpp"
+
+Match::Match(Team& t1, Team& t2, int g1, int g2, int y1, int y2, int r1, int r2, double h1, double h2)
```

Comment:
Constructor seems to be very long.

Some design question to think about:
Maybe we can introduce some other entity like TeamMatchStat or something like that. Then part of copy-paste will be eliminated and Match will contain 2 TeamStats. While Team may manage list of TeamMatchStats. That will allow to build other logic - for example for every team we can calculate some statistical data easily just based on the Team object without a need to know about Match.